packages/x-bundle/package.json

```diff
@@ -23,6 +23,17 @@
     "package:generate": "ts-node package.ts"
   },
   "dependencies": {
+    "cache-manager": "^3.6.0",
+    "cache-manager-fs-binary": "^1.0.4",
```

Comment:
isn't this poluting too much? Let's just leave redis memory mongodb and it's enough. This is an "X" bundle. And if they want custom store they can install it and use it.